extensions/guacamole-auth-cas/src/main/java/org/apache/guacamole/auth/cas/ticket/TicketValidationService.java

```diff
@@ -75,6 +84,10 @@ public String processUsername(String ticket) throws GuacamoleException {
         catch (TicketValidationException e) {
             throw new GuacamoleException("Ticket validation failed.", e);
         }
+        catch (Throwable t) {
+            logger.error("Error validating ticket with CAS server: {}", t.getMessage());
+            throw new GuacamoleInsufficientCredentialsException("Error validating ticket with CAS server.", t, CredentialsInfo.USERNAME_PASSWORD);
```

Comment:
Still not sure this is the right exception to throw.  It's the one that works - that actually displays an error on the dialog box - but still seems a little weird to throw Insufficient instead of Invalid.  However, throwing Invalid and getting the error would require some additional reworking of the index controller.